test/box/fix-memory-corruption-client.lua

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env tarantool
+
+local fiber = require('fiber')
+local net_box = require('net.box')
+
+local fibers= {}
+for i = 1, 1e2 do
+    fibers[i]=fiber.new(function()
```

Comment:
Please, put more whitespaces into the code.

Reply:
you mean local fibers= ? fixed